spring-cloud-dataflow-docs/src/main/asciidoc/applications.adoc

```diff
@@ -0,0 +1,8 @@
+[[applications]]
+= Applications
+
+[partintro]
+--
+A selection of pre-built link:http://cloud.spring.io/spring-cloud-stream-app-starters/[stream] and link:http://cloud.spring.io/spring-cloud-task-app-starters/[task/batch] starter apps for various data integration and processing scenarios facilitate learning and experimentation. For more details, review how to <<streams.adoc#spring-cloud-dataflow-register-stream-apps, register applications>>
```

Comment:
Actually, it's already there. Please see attached. 

![image](https://user-images.githubusercontent.com/1562654/28190988-a583bdfe-67e2-11e7-9054-49fafc01ee5c.png)